lib/ecmascript.ts

```diff
@@ -6180,8 +5839,8 @@ export function AddDurationToOrSubtractDurationFromZonedDateTime(
 function RoundNumberToIncrement(quantity: JSBI, increment: JSBI, mode: Temporal.RoundingMode) {
   if (JSBI.equal(increment, ONE)) return quantity;
```

Comment:
Same comment as above: why are these JSBI uses not combined into the JSBI module?